demos/README.md

```diff
@@ -1,95 +1,282 @@
+    <td><a href='node-classification/sgc/sgc-node-classification-example.ipynb'>demo</a></td>
+    <td>yes</td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><span title='Personalized Propagation of Neural Predictions'>PPNP</span></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td>yes</td>
+    <td><a href='node-classification/ppnp/ppnp-cora-node-classification-example.ipynb'>demo</a></td>
+    <td>yes</td>
+    <td>
+      <span title='UnsupervisedSampler, using link prediction'>US</span>
+      <span title='DeepGraphInfomax, using mutual information'>DGI</span>
+    </td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><span title='Approximate PPNP'>APPNP</span></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td>yes</td>
+    <td><a href='node-classification/ppnp/ppnp-cora-node-classification-example.ipynb'>demo</a></td>
+    <td>yes</td>
+    <td>
+      <span title='UnsupervisedSampler, using link prediction'>US</span>
+      <span title='DeepGraphInfomax, using mutual information'><a href='embeddings/deep-graph-infomax-cora.ipynb'>DGI</a></span>
+    </td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>GraphWave</td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td><span title='As a downstream task by training a classifier on reprentation/embedding vectors'>via RL</span></td>
+    <td><span title='As a downstream task by training a classifier on reprentation/embedding vectors'>via RL</span></td>
+    <td><a href='embeddings/graphwave-barbell.ipynb'>demo</a></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Attri2Vec</td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td>yes</td>
+    <td><a href='node-classification/attri2vec/attri2vec-citeseer-node-classification-example.ipynb'>demo</a></td>
+    <td><a href='link-prediction/attri2vec/stellargraph-attri2vec-DBLP.ipynb'>demo</a></td>
+    <td><a href='embeddings/stellargraph-attri2vec-citeseer.ipynb'>demo</a></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>GraphSAGE</td>
+    <td>see HinSAGE</td>
+    <td><a href='node-classification/graphsage/directed-graphsage-on-cora-example.ipynb'>demo</a></td>
+    <td></td>
+    <td></td>
+    <td>yes</td>
+    <td><a href='node-classification/graphsage/graphsage-cora-node-classification-example.ipynb'>demo</a></td>
+    <td><a href='link-prediction/graphsage/cora-links-example.ipynb'>demo</a></td>
+    <td>
+      <span title='UnsupervisedSampler, using link prediction'><a href='embeddings/embeddings-unsupervised-graphsage-cora.ipynb'>US</a></span>
+      <span title='DeepGraphInfomax, using mutual information'><a href='embeddings/deep-graph-infomax-cora.ipynb'>DGI</a></span>
+    </td>
+    <td><a href='node-classification/graphsage/graphsage-pubmed-inductive-node-classification-example.ipynb'>demo</a></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>HinSAGE</td>
+    <td>yes</td>
```

Comment:
Should this be "yes, nodes" (i hope i got the right column! this is meant to be the "heterogeneous" column of the hinsage row)

```suggestion
    <td>yes, nodes</td>
```

Reply:
I think HinSAGE supports different edge types. That is, it reasons about `source type --[edge type]-- target type` triples at each level of a sample, rather than just the source and target types.

https://github.com/stellargraph/stellargraph/blob/e10ab7b91a8d30a9fb21b6f475b1e4554ea6643d/stellargraph/data/explorer.py#L734-L754

Reply:
yep youre right actually :+1: